specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-06-02-preview/managedClusters.json

```diff
@@ -6663,11 +6669,15 @@
         "netNetfilterNfConntrackMax": {
           "type": "integer",
           "format": "int32",
+          "minimum": 131072,
```

Comment:
All the changes for minimum/maximum were not there before, hence they are getting reported as broken changes (even though that's bigger than the limit we enforced before)

Reply:
We could also remove these range changes to avoid breaking errors, no preference.

Reply:
We should in theory be able to get this merged without too much issue as is, IMO. But really defer to what @FumingZhang thinks/wants here. My 2c is keep them.

Reply:
Thanks Matthew! Will wait for Fuming's opinion.

Reply:
I think it's ok to have such a *breaking* change, will go through the breaking change review once I create the PR based on the dev branch later.